Source/ui_win32/Res.rc

```diff
@@ -157,6 +169,8 @@ BEGIN
         MENUITEM "State Slot",                  ID_MAIN_VM_STATESLOT
         MENUITEM "Save State\tF7",              ID_MAIN_VM_SAVESTATE
         MENUITEM "Load State\tF8",              ID_MAIN_VM_LOADSTATE
+        MENUITEM SEPARATOR
```

Comment:
@jpd002 These changed were done by the IDE itself, do you want me to reverse them?